src/chromium/crBrowser.ts

```diff
@@ -43,10 +43,26 @@ export class CRBrowser extends platform.EventEmitter implements Browser {
   private _tracingPath: string | null = '';
   private _tracingClient: CRSession | undefined;
 
-  static async connect(transport: ConnectionTransport, slowMo?: number): Promise<CRBrowser> {
+  static async connect(transport: ConnectionTransport, isPersistent: boolean, slowMo?: number): Promise<CRBrowser> {
     const connection = new CRConnection(SlowMoTransport.wrap(transport, slowMo));
     const browser = new CRBrowser(connection);
-    await connection.rootSession.send('Target.setDiscoverTargets', { discover: true });
+    const session = connection.rootSession;
+    const promises = [session.send('Target.setDiscoverTargets', { discover: true })];
+    if (isPersistent) {
```

Comment:
Let's clean this up a bit as discussed offline.

Reply:
Done.